resources/assets/js/components/HRPortal/JobIndexPage.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useState, useMemo } from "react";
+import { FormattedMessage } from "react-intl";
+import JobCard, { JobCardProps } from "../JobCard";
+import UnclaimedJobCard, { UnclaimedJobCardProps } from "../UnclaimedJobCard";
+import { JobStatus } from "../../models/lookupConstants";
+
+interface CompletedJobsAccordionProps {
+  completedJobActions: JobCardProps[];
+}
+
+const CompletedJobsAccordion: React.FC<CompletedJobsAccordionProps> = ({
+  completedJobActions,
+}) => {
+  const [isExpanded, toggleAccordion] = useState(false);
```

Comment:
While I like the idea of trying to be explicit with the naming of what the state update function does, the React convention is to name the state variable and function the same:
```js
const [isExpanded, setIsExpanded] = useState(false);
```
https://reactjs.org/docs/hooks-state.html#tip-what-do-square-brackets-mean

Makes it easier to keep track of which functions are updating what state when there are many items.